html/browsers/browsing-the-web/scroll-to-fragid/fragment-and-encoding.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<meta charset=windows-1252>
+<title>Fragment navigation: encoding</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<div style=height:10000px></div>
+<div id=&#xFF;></div>
+<div id=&#xFEFF;></div>
+<script>
+test(() => {
+  assert_equals(location.hash, "", "Page must be loaded with no hash");
+
+  location.hash = "\u00FF";
+  assert_equals(location.hash, "#%C3%BF");
+  assert_greater_than(self.scrollY, 1000, "#%C3%BF");
+
+  location.hash = "top";
+  assert_equals(self.scrollY, 0, "#top");
+
+  location.hash = "%EF%BB%BF";
+  assert_greater_than(self.scrollY, 1000, "#%EF%BB%BF");
+
+  location.hash = "%FF";
+  assert_equals(self.scrollY, 0, "#%FF");
```

Comment:
Is this correct?  self.scrollY before the hash assignment should be 1000+, and there is no element with id=U+FFFD or "%FF".  This should not change scrollY, should it?